app/models/job_proxy_dispatcher.rb

```diff
@@ -90,6 +91,36 @@ def dispatch
     _log.info "Complete - Timings: #{t.inspect}"
   end
 
+  def dispatch_container_jobs
+    jobs_to_dispatch, = Benchmark.realtime_block(:pending_container_jobs) { pending_jobs(ContainerImage) }
+    Benchmark.current_realtime[:container_jobs_to_dispatch_count] = jobs_to_dispatch.length
+
+    concurrent_container_scans_limit = Settings[:container_scanning][:concurrent_per_ems].to_i
+    if jobs_to_dispatch.length > 0
+      jobs_to_dispatch.each do |job|
```

Comment:
I suggest this because I look at the original dispatch for vm scanning and it's a long method with lots of responsibilities and therefore should only be used as reference for understanding the procedure but not for it's code quality.